### ISSUE TITLE
Support SELECT table_name.* for mix using sharding and encrypt

### ISSUE THREAD

**Issue by terrymanu** [apache/shardingsphere#4647]
If table name as owner of qualified shorthand projection for mix using sharding and encrypt, the SQL can not rewrite correctly.

SQL example

```sql
SELECT table_name.* FROM table_name
```

 should rewrite to

```sql
SELECT table_name_0.regular_col, table_name_0.cipher_col AS encrypt_col FROM table_name_0
```

**Comment by RaigorJiang** [comment]
Hi @strongduanmu 
Please see if this issue needs to be dealt with? Or postpone it?

**Comment by strongduanmu** [comment]
@RaigorJiang This issue looks similar to #11334, and i will do a test later to make sure the issue has been fixed.

**Comment by strongduanmu** [comment]
I have tested this issue and it has been fixed by PR #11334.
